datalad/distribution/tests/test_publish.py

```diff
@@ -206,7 +207,6 @@ def test_publish_with_data(origin, src_path, dst_path, sub1_pub, sub2_pub):
     # TODO: Figure out, when to fetch things in general; Alternatively:
     # Is there an option for push, that prevents GitPython from failing?
     source.repo.fetch("target")
-
```

Comment:
So there were quite a few changes in publish behavior, but none of those effects were teste since I don't see any test being added?